examples/permissioned-token/test/failing-paths.js

```diff
@@ -0,0 +1,627 @@
+const { types } = require('@algo-builder/runtime');
+const { assert } = require('chai');
+const { encodeAddress } = require('algosdk');
+const {
+  optInToASA,
+  optInToPermissionsSSC,
+  issue,
+  whitelist,
+  killToken,
+  transfer,
+  optOut,
+  forceTransfer,
+  setupEnv
+} = require('./common');
+
+const STR_TRANSFER = 'str:transfer';
+const RUNTIME_ERR1009 = 'RUNTIME_ERR1009: TEAL runtime encountered err opcode';
+const INDEX_OUT_OF_BOUND_ERR = 'RUNTIME_ERR1008: Index out of bound';
+const REJECTED_BY_LOGIC = 'RUNTIME_ERR1007: Teal code rejected by logic';
+
+describe('Permissioned Token Tests - Failing Paths', function () {
+  let runtime, master, alice, bob, elon;
+  let lsig, assetIndex, controllerAppID, permissionsAppId;
+  let asaDef, asaReserve, asaManager;
+
+  function syncAccounts () {
+    master = runtime.getAccount(master.address);
+    alice = runtime.getAccount(alice.address);
+    bob = runtime.getAccount(bob.address);
+    elon = runtime.getAccount(elon.address);
+  }
```

Comment:
let's use context, instead of hack with double `syncAccounts` hack in two files.

Reply:
yes